### PR TITLE
Update test-win11-mmd.md

### PR DESCRIPTION
**Pull request by sh783**
@VinnieFlynt @dougeby for review. Updated descriptions to remove "pre-release" references where appropriate. Note that the group still has "pre-release" in the name by design, an unfortunate artifact from when it was created prior to GA.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd.md [hunk 1]
```diff
@@ -13,23 +13,14 @@ ms.collection: M365-modern-desktop
...
 
-To enroll your devices in the pre-release test group:
-
-1. Open a new service request with the Microsoft Managed Desktop Service Engineering team.
-2. Use these values for the fields:
-    - Title: Windows 11 compatibility enrollment
-    - Request type: Change request
-    - Category: Devices
-    - Subcategory: Deployment group assignment
-3. In the description field, list the serial numbers of the devices that you want to use for Windows 11 testing. Note which, if any, of the specified devices aren't yet deployed in your Microsoft Managed Desktop tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the extra blank line. Thanks.

**Comment by yogkumgit** [comment]
@sh783 implement the suggested changes. Thanks

**Pull request merged**